app/commands/payments/github/sponsorship/handle_created.rb

```diff
@@ -0,0 +1,14 @@
+# This responds to a Github sponsorship webhook event with
+# the action being 'created'
+class Payments::Github::Sponsorship::HandleCreated
+  include Mandate
+
+  queue_as :default
+
+  initialize_with :user, :node_id, :is_one_time, :monthly_price_in_cents
+
+  def call
+    subscription = Payments::Github::Subscription::Create.(user, node_id, monthly_price_in_cents) unless is_one_time
+    Payments::Github::Payment::Create.(user, node_id, monthly_price_in_cents, subscription:)
```

Comment:
How do you find the fact that subscription is nil despite never being defined or in scope on this line. I semi-like it and semi find it very weird in Ruby! (The code is idiomatic though!)